README.rst

```diff
@@ -104,3 +130,14 @@ Apply the changes:
 You should have requested number of resources spawned on Openstack, and ansible_inventory file produced as output for Kayobe.
 
 Copy your generated id_rsa and id_rsa.pub to ~/.ssh/ on Ansible control host if you want Kayobe to automatically pick them up during bootstrap.
+
+Configure Ansible Control Host
+
+Using the `deploy-oc-networks.yml` playbook you can setup the ansible control host to include the kayobe/kayobe-config repositories with `hosts` and `admin-oc-networks`.
+It shall also setup the kayobe virtual environment, allowing for imediate configure and deployment of OpenStack.
+
+You must first run
+
+.. code-block:: console
```

Comment:
Needs a blank line here

Reply:
This comment is outdated? The UI doesn't seem to think so.